ml/backend/ggml/ggml/src/mem_dxgi_pdh.cpp

```diff
@@ -0,0 +1,304 @@
+// DXGI and PDH Performance Counters Library
+// This Windows-only (10/11) library provides accurate VRAM reporting for Intel GPUs
```

Comment:
I think you can remove the "for Intel GPUs" since this seems to work well for all Windows GPUs I've tested so far.